sql/wire/sql_api.pb.go

```diff
@@ -17,7 +16,7 @@
 		SQLRequest
 		SQLResponse
 */
-package structured
+package sql_wire
```

Comment:
`golint` doesn't like underscores in package names, so this will be a problem with `make check` as soon as we add non-generated code in this package. I'd also prefer that the package name match the directory name (do any of the linters require this?). I think it would be OK to call this `sqlwire` if you don't want just `wire` (`sql/sqlwire` stutters a bit but it seems to be recommended style following the precedent of `io/ioutil`)